kstyle/lightlystyle.cpp

```diff
@@ -4893,7 +4892,7 @@ namespace Lightly
             if( hasFocus && sunken ) textRole = QPalette::HighlightedText;
             else textRole = QPalette::WindowText;
 
-        } else if( hasFocus ) textRole = QPalette::HighlightedText;
+        } else if( option->state & State_HasFocus || sunken) textRole = QPalette::HighlightedText;
```

Comment:
Can't use `hasFocus` here because we want to change the text color when state is `State_HasFocus` AND mouseover.